.github/workflows/build_hub.yml

```diff
@@ -124,6 +125,55 @@ jobs:
           rm -f .gradle/caches/modules-2/modules-2.lock
           rm -f .gradle/caches/modules-2/gc.properties
 
+      - name: Generate New Docs
+        run: |
+          # Clean the docs before regenerating
```

Comment:
Context: This also needs the full metadata catalog, but this check will go away as soon as we migrate the schemas to the database.